examples/stun_decode.rs

```diff
@@ -0,0 +1,43 @@
+use clap::{App, Arg};
+
+use webrtc_rs_stun::message::Message;
+
+fn main() {
+    let mut app = App::new("STUN decode")
+        .version("0.1.0")
+        .author("Rain Liu <yliu@webrtc.rs>")
+        .about("An example of STUN decode")
+        .arg(
+            Arg::with_name("FULLHELP")
+                .help("Prints more detailed help information")
+                .long("fullhelp"),
+        )
+        .arg(
+            Arg::with_name("data")
+                .required_unless("FULLHELP")
+                .takes_value(true)
+                .index(1)
+                .help("base64 encoded message"),
+        );
+
+    let matches = app.clone().get_matches();
+
+    if matches.is_present("FULLHELP") {
+        app.print_long_help().unwrap();
```

Comment:
Can we add more usage info? Like what does data look like? In pion, it gives an data example: AAEAHCESpEJML0JTQWsyVXkwcmGALwAWaHR0cDovL2xvY2FsaG9zdDozMDAwLwAA